src/main/kotlin/pt/up/fe/ni/website/backend/service/AccountService.kt

```diff
@@ -24,6 +24,8 @@ class AccountService(private val repository: AccountRepository, private val enco
     fun getAccountById(id: Long): Account = repository.findByIdOrNull(id)
         ?: throw NoSuchElementException(ErrorMessages.accountNotFound(id))
 
+    fun checkAccountId(id: Long): Boolean = repository.findByIdOrNull(id) != null
```

Comment:
```suggestion
    fun doesAccountExist(id: Long): Boolean = repository.findByIdOrNull(id) != null
```
I think something like this is more explicit, wdyt?